Nos/Models/NostrConstants.swift

```diff
@@ -10,4 +10,5 @@ enum Nostr {
     /// See https://github.com/nostr-protocol/nips/blob/master/19.md
     static let profilePrefix = "nprofile"
     static let eventPrefix = "nevent"
+    static let addressPrefix = "naddr"
```

Comment:
A documentation line indicating where to find more information about naddr would be cool, just like profilePrefix did.

Reply:
Yeah, so these are all from NIP-19 and I was thinking the above comment covered it, but now I'm seeing it doesn't. I'll don't want to repeat "See NIP-19" in all of these, so I'll try to make an overall comment that covers them.